README.md

```diff
@@ -102,6 +102,11 @@ where to look for ceph configuration, user details etc.
 {
   "config_file": "/etc/ceph/ceph.conf",
   "user_id": "admin",
+  "boot_disks": [
```

Comment:
I think a more broad 'system disks' would be a better way to black list disks that shouldn't really be touched. A disk does not always have to be considered a 'boot disk' and the language behind the naming could be misleading.

Reply:
Done!